### PR TITLE
Sequelize: unnecessary calls from named getter of included model

### PR DESCRIPTION
**Pull request by NakulManchanda**
## Link to Issue
Closes: 
- Sequelize "get<model>" syntax making extra calls to database for included model

## Description of Changes
- use Object reference instead of getter to fix the issue

- We fetch Addresses with Profiles included,
- followed by profile call to each address again

Look at screenshot.

Same could be true in other places in codebase

## Test Plan
- Analyzed results
- Click around testing

image

### CODE REVIEW

File: packages/commonwealth/server/webhookNotifier.ts [hunk 1]
```diff
@@ -130,7 +130,7 @@ const send = async (models, content: WebhookContent) => {
       },
       include: [models.Profile],
     });
-    profile = await address.getProfile();
+    profile = await address.Profile;
```
Comment (NakulManchanda) [inline packages/commonwealth/server/webhookNotifier.ts:133]: how can we trigger this endpoint locally @timolegros ?

Comment (timolegros) [reply packages/commonwealth/server/webhookNotifier.ts:133]: This is a webhook, not a WebSocket so I'm not sure how to trigger this. I assume some notification types will trigger this.

Comment (timolegros) [reply packages/commonwealth/server/webhookNotifier.ts:133]:
```
  // send data to relevant webhooks
  if (
    webhook_data &&
    // TODO: this OR clause seems redundant?
    (webhook_data.chainEventType?.chain ||
      !erc20Tokens.includes(webhook_data.chainEventType?.chain))
  ) {
    await send(models, {
      notificationCategory: category_id,
      ...(webhook_data as Required<WebhookContent>),
    });
  }
```
Check `emitNotifications.ts`

Comment (NakulManchanda) [reply packages/commonwealth/server/webhookNotifier.ts:133]: if not called that often i can skip it for now, as I would like other change in getAddress to go through

Comment (NakulManchanda) [reply packages/commonwealth/server/webhookNotifier.ts]: removed

**Review by jnaviask — APPROVED** [review]

**Pull request merged**